[DJM] Logs setup guidelines on K8s (#27456)
DataDog/documentation: content/en/data_jobs/kubernetes.md

       admissionController:
         enabled: true
         mutateUnlabelled: false

>        # Optionally uncomment the 3 next lines to  enable logs collection

Reviewer: ```suggestion
       # (Optional) Uncomment the next three lines to enable logs collection
```

   Replace `<DATADOG_SITE>` with your [Datadog site][5]. Your site is {{< region-param key="dd_site" code="true" >}}. (Ensure the correct SITE is selected on the right).

   Replace `<DATADOG_AGENT_VERSION>` with version `7.55.0` or later.

>    **Optional**: Uncomment the logCollection section to start collecting application logs. This will collect all logs from all discovered containers by default. See the [Logs Collection documentation][7] for more details on the setup.

Reviewer: ```suggestion
   **Optional**: Uncomment the `logCollection` section to start collecting application logs. Once enabled, logs are collected from all discovered containers by default. See the [Kubernetes log collection documentation][7] for more details on the setup process.
```

Reviewer: I'm not sure why, but the note isn't indented like it is on the Datadog Operator tab. You might want to try indenting it by one or two more spaces to see if that fixes things.

     apm:
       portEnabled: true
       port: 8126

>      # Optionally uncomment the 3 next lines to  enable logs collection

Reviewer: ```suggestion
     # (Optional) Uncomment the next three lines to enable logs collection
```

   Replace `<DATADOG_SITE>` with your [Datadog site][4]. Your site is {{< region-param key="dd_site" code="true" >}}. (Ensure the correct SITE is selected on the right).

   Replace `<DATADOG_AGENT_VERSION>` with version `7.55.0` or later.

>   **Optional**: Uncomment the logs section to start collecting application logs. This will collect all logs from all discovered containers by default. See the [Logs Collection documentation][5] for more details on the setup.

Reviewer: ```suggestion
  **Optional**: Uncomment the logs section to start collecting application logs. Once enabled, logs are collected from all discovered containers by default. See the [Logs Collection documentation][5] for more details on the setup process.
```
